sdf.md

```diff
@@ -792,7 +791,6 @@ allowed for one of these types.
 | Quality          | Type                                                             | Description                                                        |
 |------------------+------------------------------------------------------------------+--------------------------------------------------------------------|
 | type             | "number" / "string" / "boolean" / "integer" / "array" / "object" | JSON data type (note 1)                                             |
-| enum             | array of allowed values                                          | enumeration constraint                                             |
```

Comment:
why is this removed? I thought we agreed to keep this.

Reply:
It is now in the table below this one, to keep proximity with sdfChoice.
(We could move it up again, if that is less confusing.)

Reply:
(The reason I moved it down is that it is limited to strings in SDF, and therefore compatible with, but not quite the same thing as jso `enum`.)